prime-router/metadata/hl7_mapping/common/datatype/extensionAssigningAuthority-hd.yml

```diff
@@ -0,0 +1,19 @@
+elements:
+
+  - name: assigning-authority-namespace-id
+    value: [ '%resource.extension(%`rsext-namespace-id`).value' ]
+    hl7Spec: [ '%{hl7HDField}-1' ]
```

Comment:
should there be a `condition` here?

```condition: '%resource.extension(%`rsext-namespace-id`).exists()'```

Reply:
I think there's an open style question here, since the library won't set anything if the resource is missing

Reply:
okay, I thought at first that since there are conditions for HD-2 there should be conditions on all. But I see now that HD-2 has conditions to make the two possibilities mutually exclusive